style.css

```diff
@@ -53,15 +64,15 @@ body {
   border-style: solid;
   border-width: 1px;
   color: white;
-  font-size: 3.0em;
   height: 77px;;
   margin-left: -5px;
   position: relative;
   width: 227px;
 }
 
-.calculator-output {
+#calculator-output {
```

Comment:
When you get to step 3 you'll probably want this to be a re-usable class instead of an id.